Orleans/src/Microsoft.Orleans.Host.Abstractions/IStartup.cs

```diff
@@ -7,7 +7,8 @@ namespace Microsoft.Orleans.Host.Abstractions
 {
     public interface IStartup
     {
-        IEnumerable<Type> Members { get; }
+        IEnumerable<Type> Members { get; } // not sure I understand why this?
```

Comment:
This is for the purpose of constructing any systems the host contains.  Like a silo, in this way, a silo is just another pluggable component in the service framework.  Provider are too.  IMO, this silo should not concern itself with providers, they are a framework detail.